fdbserver/fdbserver.actor.cpp

```diff
@@ -219,6 +220,7 @@ CSimpleOpt::SOption g_rgOptions[] = {
 	{ OPT_KMS_CONNECTOR_TYPE,    "--kms-connector-type",                                    SO_REQ_SEP },
 	{ OPT_KMS_REST_ALLOW_NOT_SECURE_CONECTION,  "--kms-rest-allow-not-secure-connection",      SO_NONE },
 	{ OPT_KMS_CONN_VALIDATION_TOKEN_DETAILS,     "--kms-conn-validation-token-details",     SO_REQ_SEP },
+	{ OPT_KMS_ENABLE_RECRUITMENT,                "--kms-enable-recruitment",     			SO_NONE },
```

Comment:
am not sure we want to pass this as `foundationdb.conf` param; it can be set via usual knobs?

Reply:
Sure i think we discussed last time that it may be useful as a param but im not opposed to keeping it just as a knob

Reply:
Yeah, this is more of a flexible configuration, so lets keep it as a knob.